src/classes/Globals.js

```diff
@@ -153,6 +153,21 @@ class Globals {
 		const version = parseInt(ua.browser.version.toString(), 10); // convert to string for Chrome
 		const platform = ua.os.name.toLowerCase();
 
+		// Check for Ghostery Android Browser
+		if (typeof chrome.runtime.getBrowserInfo === 'function') {
```

Comment:
It may be better to make this it's own async function that we can call in specific cases. @wlycdgr thoughts?